src/layman/rest_multipublication_test.py

```diff
@@ -16,62 +11,49 @@ class TestDeletePublicationsClass:
     authn_headers_owner = process_client.get_authz_headers(owner)
     authn_headers_deleter = process_client.get_authz_headers(deleter)
 
-    @pytest.fixture(scope="class")
+    @pytest.fixture()
```

Comment:
Why did you remove `scope="class"`?

Reply:
Reverted. Solved by setting scope="class" for ensure_layman fixture.